mizar/dp/mizar/workflows/builtins/networkpolicies/create.py

```diff
@@ -148,13 +149,32 @@ def fill_cidrs_from_directional_traffic_rules(self, data, indexed_policy_name, d
                 else:
                     data["cidrs_map_no_except"][indexed_policy_name].append(rule_item["ipBlock"]["cidr"])
             elif "namespaceSelector" in rule_item and "podSelector" in rule_item:
-                raise NotImplementedError("Not implemented")
-            elif "namespaceSelector" in rule_item:
-                raise NotImplementedError("Not implemented")
+                namespaces = kube_list_namespaces_by_labels(networkpolicy_opr.core_api, rule_item["namespaceSelector"]["matchLabels"])
```

Comment:
This should be called from the object class. similar to create_obj
https://github.com/CentaurusInfra/mizar/blob/a0c8fae4a49734c424540c3dc5d3069cce70f23e/mizar/obj/net.py#L149

https://github.com/CentaurusInfra/mizar/blob/a0c8fae4a49734c424540c3dc5d3069cce70f23e/mizar/obj/net.py#L44

Reply:
Resolved as discussed in scrum meeting.